frontend/college-toolbox/src/pages/index.astro

```diff
@@ -1,7 +1,6 @@
 ---
 import LayoutSidebar from '../app/LayoutSidebar.astro';
 import LandingReadme from '../modules/LandingReadme.astro';
-import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Update this branch with the latest changes in the main branch
`git merge main`

This line isn't present in the main branch right now